features/step_definitions/newsletter_signup_steps.rb

```diff
@@ -5,6 +5,14 @@
 When(/^I sign up to the newsletter with a valid email address$/) do
   allow(PostcodeAnywhere::EmailValidation).to receive(:valid?) { true }
 
+  # For an unknown reason the original Core::Registry::Repository[:newsletter_subscription]
+  # instance object seems to struggle to serialize. I can only guess the magic
+  # .delay method is not working as expected
+  allow(Core::Registry::Repository).to receive(:[]).with(:newsletter_subscription) do
+    Core::Repository::NewsletterSubscriptions::Fake.new
+  end
+  allow(Core::Registry::Repository).to receive(:[]).with(:category).and_call_original
```

Comment:
`Core::Registry::Repository[:category]` will call the original code. If this isn't here it will receive a message it did not expect.